src/index.ts

```diff
@@ -106,6 +108,18 @@ async function run() {
 
     // ask for snapshot of the users who were there first
     sendToZoom('/zoom/list');
+
+    // FIXME: Read in the Config file 
+    // FIXME: Schedule meetings at specified times along with an optional password for Special Commmands
+    // FIXME:    After specified Max time for the meeting elapses, end meeting if not already ended. 
+    // FIXME: When this works, don't allow a default special command password
+    //        meeting: YYYY-MM-DD-HHMM, <MaxLength HHMM>, <Meeting ID>, <Passcode>, <UserName>, <Special Command Password>
```

Comment:
Maybe use a more common format like yaml or json rather than making one up and having the order matter. Probably the easiest format would be JSON since parsing is automatic. Format might be something like:
```
{
  startTime: <ISO-standard-time-format>
  maxLength: <ISO-standard-duration-format>
  meetingID: <id>
  meetingPass: <passcode>
  meetingUsername: <username>
  specialCommandCode: <code>
}
```